app/utils/aws.ts

```
@@ -0,0 +1,463 @@
+      if (done) break;
+
+      const parsed = parseEventData(value);
+      if (!parsed) continue;
+
+      // console.log("parseEventData=========================");
+      // console.log(parsed);
+      // Handle Claude 3 models
+      if (modelId.startsWith("anthropic.claude")) {
+        if (parsed.type === "message_start") {
+          // Initialize message
+          continue;
```

Biome (1.9.4)

[error] Unnecessary continue statement

Unsafe fix: Delete the unnecessary continue statement


(lint/correctness/noUnnecessaryContinue)